content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md

```diff
@@ -27,21 +27,51 @@ To connect your `rippled` server to the XRP Testnet or Devnet, set the following
 
 2. In your `validators.txt` file:
 
-    a. Uncomment the following sections, as follows:
-
-        [validator_list_sites]
-        https://vl.altnet.rippletest.net
-
-        [validator_list_keys]
-        ED264807102805220DA0F312E71FC2C69E1552C9C5790F6C25E3729DEB573D5860
-
-    b. Comment out the following sections, as follows:
-
-        # [validator_list_sites]
-        # https://vl.ripple.com
-        #
-        # [validator_list_keys]
-        # ED2677ABFFD1B33AC6FBC3062B71F1E8397C1505E1C42C64D11AD1B28FF73F4734
+    2a. Changes to connect to the altnet
+
+        a. Uncomment the following sections, as follows to connect to the altnet:
+
+            [validator_list_sites]
+            https://vl.altnet.rippletest.net
+
+            [validator_list_keys]
+            ED264807102805220DA0F312E71FC2C69E1552C9C5790F6C25E3729DEB573D5860
+
+        b. Comment out the following sections, as follows:
+
+            # [validator_list_sites]
+            # https://vl.ripple.com
+            #
+            # [validator_list_keys]
+            # ED2677ABFFD1B33AC6FBC3062B71F1E8397C1505E1C42C64D11AD1B28FF73F4734
+        
+    2b. Changes to connect to the devnet
+    
+        a. Comment out the following sections as follows: 
+        
+            #[validator_list_sites]
+            #https://vl.altnet.rippletest.net
+
+            #[validator_list_keys]
+            ED264807102805220DA0F312E71FC2C69E1552C9C5790F6C25E3729DEB573D5860        
+        
+            # [validator_list_sites]
+            # https://vl.ripple.com
+            #
+            # [validator_list_keys]
+            # ED2677ABFFD1B33AC6FBC3062B71F1E8397C1505E1C42C64D11AD1B28FF73F4734
+        
+        b. Add the following trusted validators to the validator.txt file:
+
+            [validators]
```

Comment:
Maybe also a comment line above this clarifying that these validators are specifically for devnet? For example:

```
# Hard-coded List of Devnet Validators
[validators]
...
```